src/components/Categories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
+
+export default function Categories() {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.categories);
```

Comment:
- [ ] Clicking on the `Check status` button does result in anything. The expected `Under Construction` text does not get displayed on the page. The problem lies in this line:

  Remember that the keyword used in defining your react state is `store`. Therefore, instead of using `state`, you should use `store`.
  
  Then, in your `configureStore`, the categories reducer is defined as `checkStatus` in the `combineReducer `function. Therefore to get the status data, we need this: `store.checkStatus`.
  
  Look at the suggestion below:
  
  ```JavaScript
    const status = useSelector((store) => store.checkStatus);
  ```